docs/source/guide/build_pipelines/pipeline_basics.rst

```diff
@@ -0,0 +1,259 @@
+.. _pipeline_basics:
+
+Basics
+======
+
+Data science workflows revolve around building and refining pipelines, i.e., a series of processes that transform data into useful information/product
+(read more about pipelines :ref:`here <concepts>`).
+
+Traditionally, this is often manual and time-consuming work as data scientists (or production engineers) need to transform messy development code
+into deployable scripts for the target system (e.g., Airflow).
+
+Having the complete development process stored in artifacts, LineaPy can automate such code transformation, accelerating transition from development to production.
+
+For example, consider a simple pipeline that 1) pre-processes raw data and 2) trains a model with the pre-processed data.
+
+.. image:: pipeline.png
+  :width: 600
+  :align: center
+  :alt: Pipeline Example
+
+Pipeline Creation
+-----------------
+
+Once we have the pre-processed data and the trained model stored as LineaPy artifacts (which can be done during development sessions),
+building a pipeline reduces to “stitching” these artifacts, like so:
+
+.. code:: python
+
+   import lineapy
+
+   # Load artifacts to use in pipeline building
+   preprocessing_art = lineapy.get("iris_preprocessed")
+   modeling_art = lineapy.get("iris_model")
```

Comment:
we dont need these lines.

Reply:
This assumption is sort of inbuilt in the existing code snippet as well but if using string directly is not desirable, you can link to the section of docs that introduces these artifacts and saves them.

Reply:
Good point, making updates accordingly.